jdl/converters/jdl-to-json/jdl-to-json-field-converter.ts

```diff
@@ -98,7 +101,7 @@ function getConvertedFieldsForEntity(jdlEntity, jdlObject) {
 }
 
 function getBlobFieldData(fieldType) {
-  const blobFieldData = {
+  const blobFieldData: any = {
```

Comment:
The `any` type is used in a lot of places in the codebase. Do we need an issue for every file containing `any`?

Reply:
At first, having any types is useful (so that the project keeps working). Replacing the any types after that is mandatory. If we don't, what's the point of having TS? ;)

Reply:
Just one issue is enough

Reply:
Done: https://github.com/jhipster/generator-jhipster/issues/20037

Not sure if these kinds of vague and long-running issues are particularly useful though.

Reply:
They are, because they bring visibility to problems and the work being done to solve them :)
What you created is a good exemple of z good issue.